scripts/src/main/resources/variables.bat

```diff
@@ -13,18 +13,14 @@ set MAIN_BRANCH=main
 set SCRIPTS_PATH=scripts
 set CONF_PATH=conf
 set WORKSPACES_PATH=workspaces
+set WORKSPACESVS_PATH=workspaces_vs
 rem set SETTINGS_REL_PATH=settings
 set SETTINGS_REL_PATH=development\settings
+set ECLIPSE_TEMPLATES_REL_PATH=eclipse\workspace
 rem absolute software path
-if "%CD:~-1%" == "/" (
-  set SOFTWARE_PATH=%CD%software
-) else (
-  if "%CD:~-1%" == "\" (
-    set SOFTWARE_PATH=%CD%software
-  ) else (
-    set SOFTWARE_PATH=%CD%/software
-  )
-)
+set SOFTWARE_PATH=%CD%\software
+set DVCMDER_INSTALL_DIR=%SOFTWARE_PATH%\cmder\Cmder.exe
+set PYTHON3=%SOFTWARE_PATH%\python3\python.exe
```

Comment:
mhm, we tried to get rid of more and more variables and hardcoded tools scripting. The new way autoamtically adds all the bin folders of the tools in `software` hence it would be great if these tools could just follow convenience and best practices if possible.
Are you aware that projects customize the software folders. What if a project wants to use python 4 or no python at all or what if some project does not want to use cmder? I think we are ending up in a maintenance disaster this way. I started to escape that with the bin folder approach but I think we are not aligned on this ide project at all what is perfectly shown by the fact that a lot of scripts evolved outside github.
I think we need to do a workshop with all people that care about devon-ide or are involved in this anyhow.